isis/src/base/objs/ProcessByBrick/ProcessByBrick.cpp

```diff
@@ -660,6 +660,82 @@ void ProcessByBrick::SetOutputRequirements(int outputRequirements) {
     omgrs.clear();
   }
 
+
+  /**
+   * Starts the systematic processing of the input cube by moving an arbitrarily-shaped
+   * brick through the cube. This method allows multiple input and output
+   * cubes.
+   *
+   * @deprecated Please use ProcessCubeInPlace, ProcessCube, or ProcessCubes
```

Comment:
Are we adding deprecated functionality here?

Reply:
These deprecated tags need to be removed, the functional signatures are brand new.